src/shapes/polyline.class.ts

```diff
@@ -255,8 +255,7 @@ export class Polyline extends FabricObject {
       changed &&
       (((key === 'scaleX' || key === 'scaleY') &&
         this.strokeUniform &&
-        this.strokeBBoxAffectingProperties.includes('strokeUniform') &&
-        this.strokeLineJoin !== 'round') ||
```

Comment:
I noticed that the changes made to the projection of the `round` case made it also necessary to recalculate the projection when `strokeUniform = true`

Reply:
fixed at https://github.com/fabricjs/fabric.js/pull/8374/commits/0810190790ff202b8ad228f6e200068550b11e42